docs/self-hosted/docker-guide.md

```diff
@@ -83,36 +83,27 @@ project:
 version: '3'
 services:
   database:
-    container_name: database
     image: postgis/postgis:13-master
```

Comment:
Time to update this to v15? 🤔

Reply:
Yeah was thinking the same because 13 is getting old quick. But didnt feel like a requirement for this PR

Reply:
Yes, let's keep that for another PR 👍